Package@swift-5.5.swift

```diff
@@ -0,0 +1,47 @@
+// swift-tools-version:5.5
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the DeviceKit open source project
+//
+// Copyright © 2014 - 2018 Dennis Weissmann and the DeviceKit project authors
+//
+// License: https://github.com/dennisweissmann/DeviceKit/blob/master/LICENSE
+// Contributors: https://github.com/dennisweissmann/DeviceKit#contributors
+//
+//===----------------------------------------------------------------------===//
+
+import PackageDescription
+
+let package = Package(
+    name: "DeviceKit",
+    platforms: [
+        .iOS(.v9),
+        .tvOS(.v9),
+        .watchOS(.v2)
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "DeviceKit",
+            targets: ["DeviceKit"]
+        ),
+        .library(
+            name: "DeviceKitDynamic",
```

Comment:
added DeviceKitDynamic, everything else is 1:1 copy from Package.swift

Reply:
Have you tested this already in Production?